ietf-layer0-types-ext.yang

```diff
@@ -648,10 +722,18 @@ module ietf-layer0-types-ext {
         base line-coding;
       }
       config false;
-      description "Bit rate/line coding of optical tributary signal";
+      description
+        "Bit rate/line coding of the optical tributary signal.";
       reference 
         "ITU-T G.698.2 section 7.1.2";
     }
+    leaf bitrate {
+      type uint16;

```

Comment:
shouldn't this be a decimal value ?
or maybe this value refers to the gross bitrate: eg 100, 200, ....

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/36/commits/ff8a58466944a4ad32b0c506256f83c5713ee7de